src/NServiceBus.Metrics.PerformanceCounters/PerformanceCountersFeature.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.Features;
+
+class PerformanceCountersFeature : Feature
+{
+    public PerformanceCountersFeature()
+    {
+        Defaults(s =>
+        {
+            options = s.EnableMetrics();
+            s.SetDefault(UpdateIntervalKey, TimeSpan.FromSeconds(2));
+        });
+    }
+
+    protected override void Setup(FeatureConfigurationContext context)
+    {
+        var logicalAddress = context.Settings.LogicalAddress();
+
+        var legacyInstanceNameMap = new Dictionary<string, CounterInstanceName?>
```

Comment:
This mapping logic is also implemented in the script generation. Decided to keep it redundant to not have unnecessary references